app/components/organisms/ArticleEditorV2.vue

```diff
@@ -7,7 +7,7 @@
       spellcheck="false"
       maxlength="255"
       @input="onInputTitle"
-      @keydown.enter.prevent
+      @keydown.enter="handleEnter"
```

Comment:
処理的に可能であれば、`prevent`は利用したままのほうが良いと思いました。

```suggestion
      @keydown.enter.prevent="handleEnter"
```